Source/CarthageKit/Xcode.swift

```diff
@@ -1012,9 +1012,35 @@ public func stripDSYM(_ dSYMURL: URL, keepingArchitectures: [String]) -> SignalP
 
 /// Strips a universal file from unexpected architectures.
 private func stripBinary(_ binaryURL: URL, keepingArchitectures: [String]) -> SignalProducer<(), CarthageError> {
-	return architecturesInPackage(binaryURL)
-		.filter { !keepingArchitectures.contains($0) }
-		.flatMap(.concat) { stripArchitecture(binaryURL, $0) }
+  let fileManager = FileManager().reactive
```

Comment:
Should this use `FileManager.default`?

Reply:
I don't remember the specific details, but there is (was?) a concurrency issue with the default FileManager.

Ah, here it is:

https://useyourloaf.com/blog/nsfilemanager-defaultmanager-is-not-thread-safe/

and once again, I know things that are no longer true. :-) I'll update it.

Reply:
TIL!